### DISCUSSION TITLE
Crash when importing  music library for first time

### DISCUSSION THREAD

**Discussion by dbuttric** [beetbox/beets#4075] [Q&A]
I started the import run for the first time, and it got about ten albums in, and this happened.
The import will not proceed past this problem.

[beet crash.txt](https://github.com/beetbox/beets/files/7230108/beet.crash.txt)

**Comment by sampsyo** [comment]
That looks bad! Could you please report the traceback to [the MediaFile repository](https://github.com/beetbox/mediafile)? The bug seems to be there, and we should definitely fix it.
